cni-plugins/sriov-passthrough-cni/plugin/sriov-passthrough-cni

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+readonly IP=169.255.0.0
+readonly SUBNET_MASK=24
+readonly GW_IP=169.255.0.1
+readonly CONTAINER_NETNS_LINK="/var/run/netns/$CNI_CONTAINERID"
+readonly LOGFILE=/var/log/pfcni.log
+
+function main() {
+    case "$CNI_COMMAND" in
+        ADD)
+            log "Received ADD command. CNI_ARGS: $CNI_ARGS"
+            setup_netns
+            # add_veth_pair
+            flip_vfs_to_default_drivers
+            setns_sriov_ifs
+            gen_result_config
+            ;;
+        DELETE)
```

Comment:
The command is "DEL": https://github.com/containernetworking/cni/blob/master/SPEC.md